build/ublue-os-just/ublue-os-just.spec

```diff
@@ -49,20 +53,31 @@ mkdir -p -m0755  %{buildroot}%{_bindir}
 install -Dm755 %{SOURCE9} %{buildroot}%{_bindir}/ujust
 install -Dm755 %{SOURCE10} %{buildroot}%{_bindir}/ugum
 
+# Add bash library for use in just
+mkdir -p -m0755 %{buildroot}/usr/lib/ujust/
+install -Dm644 %{SOURCE12} %{buildroot}/usr/lib/ujust
+install -Dm644 %{SOURCE13} %{buildroot}/usr/lib/ujust
+install -Dm644 %{SOURCE14} %{buildroot}/usr/lib/ujust
+install -Dm644 %{SOURCE15} %{buildroot}/usr/lib/ujust
```

Comment:
Apologies, I know that I previously suggested `%{buildroot}/usr/lib/ujust` but I realized hard-coding `/usr` breaks the concept of exec-prefix. And while it probably would work in this situation, it's not ideal, and it's nice when we can do things "right".

I found a helpful reference which confirmed my minor concern: https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/

So, without more information about "best practice",  I think I'm indifferent on if we use:
- `%{buildroot}/%{_exec_prefix}/lib/ujust`
-  or `%{buildroot}/%{_datadir}/ujust`

 but i do think either of these is better than fully hard-coding `/usr/lib/ujust` in the spec file.